src/main/java/core/basesyntax/service/RegistrationService.java

```diff
@@ -1,7 +1,8 @@
 package core.basesyntax.service;
 
+import core.basesyntax.exceptions.ValidationException;
 import core.basesyntax.model.User;
 
 public interface RegistrationService {
-    User register(User user);
+    User register(User user) throws ValidationException;
```

Comment:
What do you think should be done using your method? In this case you don't need to catch anything, just send incorrect data and throw RuntimeException
```suggestion
    User register(User user);
```